README.rst

```diff
@@ -16,8 +16,7 @@ Checkoutmanager works on linux, osx and windows.
 Starting to use it
 ------------------
 
-Starting is easy.  Just ``easy_install checkoutmanager`` and run
-``checkoutmanager``.
+Starting is easy.  Just ``pip checkoutmanager`` and run ``checkoutmanager``.
```

Comment:
That should be pip **install** checkoutmanager.

Reply:
Oopsie!
Thanks, I've fixed it in 2.6.1. This just looks too ridiculous to leave as-is on pypi :-)